### ISSUE TITLE
Does the  architecture of Tiny NASNet-A in this code??? 

### ISSUE THREAD

**Issue by ReverseSystem001** [XiaohangZhan/cdp#17]
Does the  architecture of Tiny NASNet-A in https://github.com/XiaohangZhan/face_recognition_framework/blob/master/models/backbones/nasnet.py, and the NASNetASmall in the git link is the same with Tiny NASNet-A, right?

**Comment by XiaohangZhan** [comment]
Yes you are right

**Comment by ReverseSystem001** [comment]
thanks for your quickly replay. can you tell me the training details of Tiny NASNet-A? the epoch, iterations,and learning rate.

**Comment by XiaohangZhan** [comment]
Please see https://arxiv.org/pdf/1809.01407.pdf Appendix A1 and A3. For Tiny NASNet-A, for supervised initialization (1/11 data), lr is 0.1, batch size is 256, M is 50; for joint training (full data), please see A3

**Comment by ReverseSystem001** [comment]
thanks.
